examples/cis3-nft-sponsored-txs/src/lib.rs

```diff
@@ -1160,35 +1085,35 @@ pub struct PublicKeyQuery {
 ///
 /// It rejects if:
 /// - It fails to parse the parameter.
-#[receive(
-    contract = "cis3_nft",
-    name = "publicKeyOf",
-    parameter = "ContractPublicKeyQueryParams",
-    return_value = "ContractPublicKeyQueryResponse",
-    error = "ContractError"
-)]
-fn contract_public_key_of<S: HasStateApi>(
-    ctx: &impl HasReceiveContext,
-    host: &impl HasHost<State<S>, StateApiType = S>,
-) -> ContractResult<ContractPublicKeyQueryResponse> {
-    // Parse the parameter.
-    let params: ContractPublicKeyQueryParams = ctx.parameter_cursor().get()?;
-    // Build the response.
-    let mut response: Vec<Option<(PublicKeyEd25519, u64)>> =
-        Vec::with_capacity(params.queries.len());
-    for query in params.queries {
-        // Query the state for the public_key and nonce.
-        let entry = host
-            .state()
-            .public_key_registry
-            .get(&query.account)
-            .map(|registry_entry| (registry_entry.0, registry_entry.1));
-
-        response.push(entry);
-    }
-    let result = ContractPublicKeyQueryResponse::from(response);
-    Ok(result)
-}
+// #[receive(
+//     contract = "cis3_nft",
+//     name = "publicKeyOf",
+//     parameter = "ContractPublicKeyQueryParams",
+//     return_value = "ContractPublicKeyQueryResponse",
+//     error = "ContractError"
+// )]
+// fn contract_public_key_of<S: HasStateApi>(
```

Comment:
What is the intention here?

Reply:
I replaced this function with the `publicKeyOf` function and the `nonceOf` function.